units/host_termination.go

```diff
@@ -390,15 +390,6 @@ func (j *hostTerminationJob) checkAndTerminateCloudHost(ctx context.Context, old
 	}
 
 	if cloudInfo.Status == cloud.StatusTerminated {
-		grip.Warning(message.Fields{
-			"message":      "attempted to terminate an already terminated host",
-			"state_reason": cloudInfo.StateReason,
```

Comment:
I think it's okay to remove this but I recently did some work to add this attribute right here (it was merged after the outages) to see what it says during the outages. Right now, it's all giving the same reason. The ticket is: https://jira.mongodb.org/browse/DEVPROD-5582

Reply:
That sounds fine to me. Perhaps we can just format it into the message at L404